examples/asr/conf/quartznet_15x5_zh.yaml

```diff
@@ -0,0 +1,484 @@
+name: &name "QuartzNet15x5"
+
+model:
+  sample_rate: &sample_rate 16000
```

Comment:
You could refactor same as above, or keep it as here. I still advice changing the `cls` and `params` part since it will be deprecated in later versions of hydra.